### PR TITLE
Update everything needed for the fast-jar switch

### PR DESCRIPTION
**Pull request by gsmet**
FWIW, the quick and dirty script updating the dockerfiles:
```
#! /bin/bash

pushd quarkus-quickstarts

for quickstart in `find . -mindepth 1 -maxdepth 1 -not -path '*/\.*' -type d | sort`; do
        if [ $quickstart != './funqy-quickstarts' ] && [ $quickstart != './gradle' ] && [ -f "$quickstart/pom.xml" ]; then
        	for dockerfile in `ls /tmp/getting-started/src/main/docker/`; do
                        mkdir -p $quickstart/src/main/docker
			cp -f /tmp/getting-started/src/main/docker/$dockerfile $quickstart/src/main/docker/
               		quickstartname=${quickstart:2}
                	projectname=${quickstartname%-quickstart}
                	sed -i "s/getting-started/${projectname}/g" $quickstart/src/main/docker/$dockerfile
        	done
	fi
done

popd
```

### CODE REVIEW

**Comment by gsmet** [comment]
@famod could you check the PR? I think just taking a few samples and checking things are correct is enough.

File: funqy-quickstarts/src/main/docker/Dockerfile.jvm [hunk 1]
```diff
@@ -0,0 +1,54 @@
+####
```
Comment (famod) [inline funqy-quickstarts/src/main/docker/Dockerfile.jvm:1]: This one slipped though (there is a typo in your script; `funky` -> `funqy`).

**Review by famod — CHANGES_REQUESTED** [review]
I took quite a few samples and LGTM, except this one superfluous set of files in `funqy-quickstarts`.

It wouldn't hurt to have another pair of reviewing eyes, though.

File: funqy-quickstarts/src/main/docker/Dockerfile.jvm (hunk 1 above)
Comment (gsmet) [reply funqy-quickstarts/src/main/docker/Dockerfile.jvm:1]: Raaaaaah. I fixed it manually first and then automated it but did it wrong :).

**Comment by gsmet** [comment]
@geoand any chance you could review a few files too?

File: funqy-quickstarts/src/main/docker/Dockerfile.jvm (hunk 1 above)
Comment (gsmet) [reply funqy-quickstarts/src/main/docker/Dockerfile.jvm]: Fixed!

**Comment by geoand** [comment]
I'll try :)

**Comment by geoand** [comment]
Any reason why `openssl` was removed from the docker files?

File: amazon-dynamodb-quickstart/src/main/docker/Dockerfile.native [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
+# ./mvnw package -Pnative
 #
 # Then, build the image with:
 #
-# docker build -f src/main/docker/Dockerfile.native -t quarkus/dynamodb-client .
+# docker build -f src/main/docker/Dockerfile.native -t quarkus/amazon-dynamodb .
 #
 # Then run the container using:
 #
-# docker run -i --rm -p 8081:8081 quarkus/dynamodb-client
```
Comment (geoand) [inline amazon-dynamodb-quickstart/src/main/docker/Dockerfile.native:14]: I wonder if there is a reason why `8081` was being used

**Comment by gsmet** [comment]
Well, they are just a copy of the Quarkus Dockerfiles generated for a new project.

**Comment by geoand** [comment]
> Well, they are just a copy of the Quarkus Dockerfiles generated for a new project.

OK, then I assume it's correct :)

**Comment by gsmet** [comment]
I have a vague recollection that it's now compiled statically in the native executable.

File: amazon-dynamodb-quickstart/src/main/docker/Dockerfile.native (hunk 2 above)
Comment (gsmet) [reply amazon-dynamodb-quickstart/src/main/docker/Dockerfile.native:14]: Yeah, no idea :).

The README talks about 8080 so I think we should be good.

**Comment by gsmet** [comment]
I added a commit to exclude `target/quarkus-app/` from .dockerignore

**Comment by geoand** [comment]
Good idea

**Review by geoand — APPROVED** [review]
This is awesome! Thanks so much!

**Review by famod — APPROVED** [review]
👍

**Pull request merged**